<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxelVolumeView</class>
 <widget class="QWidget" name="VoxelVolumeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QSlider" name="verticalSlider">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>slice:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_L_slice">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>48</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="WindowingWidget" name="_W_windowing" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Zoom factor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>335</width>
        <height>204</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="_L_image">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="_SB_zoom">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="minimum">
      <double>0.250000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.250000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WindowingWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/windowingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VoxelVolumeView</receiver>
   <slot>updateImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_zoom</sender>
   <signal>valueChanged(double)</signal>
   <receiver>VoxelVolumeView</receiver>
   <slot>updateImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateImage()</slot>
 </slots>
</ui>
